ui/services/config.js

```diff
@@ -95,6 +95,13 @@ export const setUseMock = ({ parameterContext }) => {
   }
 }
 
+export const setDisplayMap = ({ parameterContext }) => {
+  let displayMap = getValueFromPath("displayMap")
+  if(displayMap!==null) {
+    parameterContext.setDisplayMap(displayMap==="false"?false:true)
```

Comment:
`displayMap` can never be `null` according to the code, it's initialized to `true` and can only be set to `false`, I don't think the check is necessary, I would have removed the null check and went like :`parameterContext.setDisplayMap(JSON.parse(displayMap))`